ArduPlane/servos.cpp

```diff
@@ -206,10 +206,12 @@ void Plane::flaperon_update()
       percentage of flaps. Flap input can come from a manual channel
       or from auto flaps.
      */
-    float aileron = SRV_Channels::get_output_scaled(SRV_Channel::k_aileron);
-    float flap_percent = SRV_Channels::get_slew_limited_output_scaled(SRV_Channel::k_flap_auto);
-    float flaperon_left  = constrain_float(aileron + flap_percent * 45, -4500, 4500);
-    float flaperon_right = constrain_float(aileron - flap_percent * 45, -4500, 4500);
+    const float aileron = SRV_Channels::get_output_scaled(SRV_Channel::k_aileron);
+    const float flap_percent = SRV_Channels::get_slew_limited_output_scaled(SRV_Channel::k_flap_auto);
+    const float elevator_hf = 0.01f * (float)g.hf_elev_flap_mix_gain_pct *  SRV_Channels::get_output_scaled(SRV_Channel::k_elevator_hf);
+    const float elevator_lf = 0.01f * (float)g.lf_elev_flap_mix_gain_pct *  SRV_Channels::get_output_scaled(SRV_Channel::k_elevator_lf);
+    const float flaperon_left  = constrain_float(aileron + elevator_hf + elevator_lf + flap_percent * 45, -4500, 4500);
```

Comment:
maybe we should have a HF_ELEV_OPTIONS with enable for mixing into flaperons? also, should we separately enable mixing the HF and LF components?